src/shared/models/custom/ModelUtils.ts

```diff
@@ -20,4 +22,26 @@ export class ModelUtils {
   static generateSecondarySponsorOrganizationId(value: string) {
     return `${value}-secondary-sponsor`
   }
+
+  static identifyAssigner(regulationCode: string, qualification: string): AssignerForSecondaryIdentifier {
```

Comment:
```suggestion
  static identifyAssigner(regulationCode: string, qualification?: string): AssignerForSecondaryIdentifier {
```
Comme ça, tu peux enlever le undefined dans la factory CTIS